src/ir/module-splitting.cpp

```diff
@@ -334,6 +335,7 @@ struct ModuleSplitter {
     exportImportCalledPrimaryFunctions();
     setupTablePatching();
     shareImportableItems();
+    removeUnusedElements();
```

Comment:
If this ran on the primary then it would run after each of the N splits in MultiSplit mode, which would be slow - but it looks like you only run it on the secondary? If so maybe rename this to reflect that?

I guess it's not possible the primary can also benefit from this? (assume the module is already optimized)

Reply:
Right, everything besides functions is exported from the primary module, so there would be no removable module elements.

I'll clarify the method name.